.ci/CI/src/modules/IntegTest.jl

```diff
@@ -321,7 +321,10 @@ function add_integration_test_job_yaml!(
             generate_job_yaml!(
                 p,
                 test_package,
-                target_branch,
+                # TODO: `dev` is the "default" branch
+                # a possible, different branch is stored in package_info
+                # simplify the interface
+                "dev",
```

Comment:
Does this restrict the intended functionality, because it limits the "target_branch" to "dev"?